ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersPage.tsx

```diff
@@ -67,9 +108,40 @@ function ProfileClustersPage() {
                     handleToggleChange={handleProfilesToggleChange}
                 />
             </PageSection>
-            <PageSection variant="default">
-                <PageSection variant="light" component="div">
-                    <Title headingLevel="h2">Profile results</Title>
+            <PageSection variant="default" className="pf-v5-u-py-0">
+                <PageSection variant="light" className="pf-v5-u-p-0">
```

Comment:
I think I like this styling better but I wanted to point out that we should try to standardize this. In some parts of the app, there's padding on the `<section>` where a table is being use, but in other parts we don't include the padding.

Also Mark had pointed out to me the `component` prop on `PageSection`, semantically speaking this might not make sense to have the second nested `<section>` element in the html here. I was just using it for padding purposes previously.

Reply:
Maybe I'll look into doing a Design Doc for this. Maybe we could collaborate on that.